spec/chromium-spec.js

```diff
@@ -937,6 +937,23 @@ describe('chromium feature', () => {
   })
 
   describe('storage', () => {
+    describe('DOM storage quota override', () => {
+      ['localStorage', 'sessionStorage'].forEach((storageName) => {
+        it(`can store at least 50MiB in ${storageName}`, (done) => {
+          assert.doesNotThrow(() => {
+            const storage = window[storageName]
+            const testKeyName = '_electronDOMStorageQuotaOverrideTest'
+            // 25 * 2^20 UTF-16 characters will require 50MiB
+            const arraySize = 25 * Math.pow(2, 20)
+            storage[testKeyName] = new Array(arraySize).fill('X').join('')
+            assert.strictEqual(storage[testKeyName].length, arraySize)
```

Comment:
We are in the process of moving away from `assert`s in favour of [Chai](https://www.chaijs.com)'s `expect`s.
Can you please replace this line with the following?
```suggestion
            expect(storage[testKeyName])to.have.lengthOf(arraySize)
```

Reply:
Sure -- I had just attempted to copy the conventions from a nearby test.